Apps/Admin/Server/Services/IDashboardService.cs

```diff
@@ -62,5 +64,15 @@ public interface IDashboardService
         /// <param name="timeOffset">The offset from the client browser to UTC.</param>
         /// <returns>A dictionary pairing the ratings with the counts.</returns>
         IDictionary<string, int> GetRatingSummary(string startPeriod, string endPeriod, int timeOffset);
+
+        /// <summary>
+        /// Retrieves year of birth counts for users that have logged in between two dates.
+        /// </summary>
+        /// <param name="startPeriod">The period start of last login of users.</param>
```

Comment:
NIT:
The start of the period since users' last login
The end of the period since users' last login

Reply:
Updated to the same wording that's used in other areas.